scripts/mixins/families/chigoe.lua

```diff
@@ -20,6 +34,22 @@ g_mixins.families.chigoe = function(chigoeMob)
         mob:hideName(true)
         mob:setUntargetable(true)
     end)
+
+    chigoeMob:addListener('CRITICAL_TAKE', 'CHIGOE_CRITICAL_TAKE', function(mob)
+        mob:setHP(0)
```

Comment:
With these, isn't it also the case that chigoes award no xp?

Reply:
iirc clay is right and chigs that die via the insta kill mechanic do not, but if they are killed normally they will still award exp (if in level range to do so)

this puts us in the awkward spot of needing to apply the -exp mod just before killing them in this section, rather than as a general edit to the mob as is happening [on line 89 of the chigoe_pet mixin](https://github.com/LandSandBoat/server/pull/4381/files#diff-d61b80f03fdd22cdb3c9647e768696cc310e4a396a2b7b15f5df9d7b0e107397R89)

Reply:
confirmed coz I just got limit points off the ones i didn't use the special mechanism for (blizaga'd them) and didn't one the ones I stepped on (pun intended)

Reply:
i moved the mod inside the one-shot listeners